validation/shallow_water_model/shallow_water_jet_topography.jl

```diff
@@ -0,0 +1,167 @@
+using Oceananigans
+using Oceananigans.Models: ShallowWaterModel
+   
+Nx, Ny = 50, 128
+Lx, Ly = 2π,   10
+H₀     = 10
+α, β   = 0.1,  0.1
+Lb, Hb = α * Ly, β * H₀
+
+f, g, U  = 1, 1, -1.0
+Δη = f * U / g 
+
+grid = RectilinearGrid(size = ( Nx, Ny),
+                          x = (  0, Lx),
+                          y = (-Ly, Ly),
+                   topology = (Periodic, Bounded, Flat))
+
+η(x, y) = Δη * tanh(y)
+b(x, y) = Hb * tanh(y/Lb)
+
+h̄(x, y) = H₀ + η(x, y) + b(x, y)
+ū(x, y) = - U * sech(y)^2
+#ωb(x, y) = - 2 * U * sech(y)^2 * tanh(y)
+
+small_amplitude = 1e-4
+ 
+ uⁱ(x, y) = ū(x, y) + small_amplitude * exp(-y^2) * randn()
+uhⁱ(x, y) = uⁱ(x, y) * h̄(x, y)
+
+ū̄h(x, y) = ū(x, y) * h̄(x, y)
+
+bathymetry(x,y) = -b(x,y)
```

Comment:
why we define something that is `b(x, y)` and then `bathymetry = - b`? what is `b` then? let's define the bathymetry from the beginning?

Reply:
This is not direct and should be fixed.  Agreed!